### DISCUSSION TITLE
1.2.1混合打包后，安装后，点击登录没有响应，beta版本没有类似问题

### DISCUSSION THREAD

**Discussion by bjfk2006** [apache/streampark#726] [Q&A]
![image](https://user-images.githubusercontent.com/6290119/154634104-19b38468-2e83-4835-8f26-c433775e02ac.png)

**Comment by bjfk2006** [comment]
是不是要修改vi streamx/streamx-console/streamx-console-webapp/.env.production，每次更换服务器都要重新打包编译？

**Reply by zhouhongyu888** [reply]
是的，这个问题我也遇到了，每次都要更改.env.production后再编译，否则就会报跨域的错误

**Reply by Cloud-ink** [reply]
请问下你们开发的话是在哪里调试的啊，我看开发文档里写的是idea可以启动，这个idea是安装在linux上的么？
